ui/shared/utils/panelAppUtils.js

```diff
@@ -0,0 +1,54 @@
+export const moiToMoiInitials = (rawMoi) => {
+  if (!rawMoi) {
+    return []
+  }
+
+  const moi = rawMoi.toUpperCase()
+
+  if (moi.startsWith('MONOALLELIC')) {
+    if (moi.includes('PATERNALLY IMPRINTED')) {
+      return []
+    }
+    if (moi.includes('MATERNALLY IMPRINTED')) {
+      return []
+    }
+    return ['AD']
+  }
+  if (moi.startsWith('X-LINKED') || moi.startsWith('X LINKED')) {
+    if (moi.includes('BIALLELIC MUTATIONS')) {
+      return ['XR']
+    }
+    if (moi.includes('MONOALLELIC MUTATIONS')) {
+      return ['XR', 'XD']
+    }
+  }
+  if (moi.startsWith('BIALLELIC')) {
+    return ['AR']
+  }
+  if (moi.startsWith('BOTH')) {
+    return ['AD', 'AR']
+  }
+  if (moi.startsWith('MITOCHONDRIAL')) {
```

Comment:
all if conditionals from here down are unnecessary, you should just use the fallback `return []`

Reply:
Ok